### DISCUSSION TITLE
第4节课笔记(19班)

### DISCUSSION THREAD

**Discussion by LYMDLUT** [InternLM/Tutorial#252] [笔记]
第4节课笔记(19班)

**Comment by Sixdes** [comment]
https://bbs.csdn.net/topics/617921853

**Comment by Jiangzzzzl** [comment]
https://blog.csdn.net/2301_80618119/article/details/135527634?spm=1001.2014.3001.5501

**Comment by hu-qi** [comment]
https://zhuanlan.zhihu.com/p/677770587

**Comment by romanwin28** [comment]
https://blog.csdn.net/George_han/article/details/135589213

**Comment by jts250** [comment]
https://zhuanlan.zhihu.com/p/677167323

**Comment by JankinTian** [comment]
https://blog.csdn.net/xiao_xian_/article/details/135612476

**Comment by melonwine** [comment]
https://github.com/melonwine/InternLMPlay/blob/master/04/note04.md

**Comment by AllYoung** [comment]
https://blog.csdn.net/sunshine_youngforyou/article/details/135635903

**Comment by mkdirmushroom** [comment]
[第4节课笔记](https://github.com/mkdirmushroom/note/blob/main/InternLM/tutorial3_note.md)

**Comment by Hamasuta666** [comment]
第4节课笔记（作业在文末）：https://zhuanlan.zhihu.com/p/680582728
